src/apiary.apib

```diff
@@ -67,6 +67,8 @@ HOST: https://api.teamleader.eu
 
 :[Milestones](./07-projects/milestones.apib)
 
+:[Project Items](./07-projects/project-items.apib)
```

Comment:
As discussed we could  for now **not** include project-items in apiary.apib and merge this documentation.